src/test/java/core/basesyntax/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,261 @@
+        assertEquals(currentUser1, user1);
+        assertEquals(currentUser2, user2);
+        assertTrue(actual);
+
+    }
+
+    @Test
+    void check_userAgeIsMoreThanEighteen_Ok() {
+        user1 = new User(35624576L, "MadGeek", "11sds11", 122);
+        user2 = new User(19358745L, "FlyBoys", "0dsdfsdf", 28);
+        user3 = new User(66354214L, "MilkyWilki", "absssc", 45);
+        user4 = new User(33576954L, "Pokemon", "qqqddww", 19);
+        user5 = new User(79865341L, "AngryDad", "//vdfffd", 60);
+        User currentUser1 = registrationService.register(user1);
+        User currentUser2 = registrationService.register(user2);
+        User currentUser3 = registrationService.register(user3);
+        User currentUser4 = registrationService.register(user4);
+        User currentUser5 = registrationService.register(user5);
+        final boolean actual = currentUser1.getAge() > EIGHTEEN_YEARS_OLD
+                && currentUser2.getAge() > EIGHTEEN_YEARS_OLD
+                && currentUser3.getAge() > EIGHTEEN_YEARS_OLD
+                && currentUser4.getAge() > EIGHTEEN_YEARS_OLD
+                && currentUser5.getAge() > EIGHTEEN_YEARS_OLD;
+        assertEquals(currentUser1, user1);
+        assertEquals(currentUser2, user2);
+        assertEquals(currentUser3, user3);
+        assertEquals(currentUser4, user4);
+        assertEquals(currentUser5, user5);
+        assertTrue(actual);
+    }
+
+    @Test
+    void check_userLoginIsLessThanSix_notOk() {
+        user1 = new User(35624576L, "Mad", "11sds11", 22);
+        user2 = new User(19358745L, "F", "0dsdfsdf", 28);
+        user3 = new User(66354214L, "Milky", "absssc", 45);
+        user4 = new User(33576954L, "Po", "qqqddww", 19);
+        user5 = new User(79865341L, "Angr", "//vdfffd", 60);
+        assertThrows(InvalidDataException.class, () -> registrationService.register(user1));
+        assertThrows(InvalidDataException.class, () -> registrationService.register(user2));
+        assertThrows(InvalidDataException.class, () -> registrationService.register(user3));
+        assertThrows(InvalidDataException.class, () -> registrationService.register(user4));
+        assertThrows(InvalidDataException.class, () -> registrationService.register(user5));
+    }
+
+    @Test
+    void check_userLoginIsSixLength_Ok() {
+        user1 = new User(35624576L, "Mad123", "11sds11", 22);
+        user2 = new User(19358745L, "F12345", "0dsdfsdf", 28);
+        User currentUser1 = registrationService.register(user1);
+        User currentUser2 = registrationService.register(user2);
+        boolean actual = currentUser1.getLogin().length() == DEFAULT_LENGTH_FOR_LOGIN_AND_PASSWORD
+                && currentUser2.getLogin().length() == DEFAULT_LENGTH_FOR_LOGIN_AND_PASSWORD;
+        assertEquals(currentUser1, user1);
+        assertEquals(currentUser2, user2);
+        assertTrue(actual);
+
+    }
+
+    @Test
+    void check_userLoginIsMoreThanSixLength_Ok() {
```

Comment:
```suggestion
    void check_userLoginIsMoreThanSixLength_Ok() {
```
use lowercase after underscore - `_ok`
try to name methods without hardcoding limits, which may change later - and you will need to change method names too

Reply:
okay, i learnd about name without hardcoding limits and try next time

Reply:
++